### PR TITLE
Add option to override the title of the event archive portlets.

### PR DESCRIPTION
**Pull request by mbaechtold**

### CODE REVIEW

File: ftw/contentpage/locales/de/LC_MESSAGES/ftw.contentpage.po [hunk 1]
```diff
@@ -254,6 +258,36 @@ msgstr "Geben Sie den Alternativtext für das Bild ein. Der Alternativtext richt
 msgid "description_image_clickable"
 msgstr "Wird das Bild angeklickt, öffnet es sich in einem Overlay"
 
+#. Default: "Renders a widget containing the number of events by year and month."
+#: ./ftw/contentpage/portlets/event_archive.py:83
+msgid "event_archive_portlet_add_form_description"
+msgstr "Zeigt die Anzahl Events gruppiert nach Jahr und Monat an."
```
Comment (jone) [inline ftw/contentpage/locales/de/LC_MESSAGES/ftw.contentpage.po]: `Events` ist englisch - gibts da eine deutsche Variante?

**Comment by mbaechtold** [comment]
@jone, should we remove support for Plone 4.1?

**Comment by jone** [comment]
@mbaechtold 👍 you may drop Plone 4.1 support

**Comment by mbaechtold** [comment]
@jone, I have remove support for Plone 4.1.

**Pull request merged**